apps/trustlab/src/components/Page/Page.test.js

```diff
@@ -0,0 +1,16 @@
+import { createRender } from "@commons-ui/testing-library";
+
+import Page from "./Page";
+
+import theme from "@/trustlab/theme";
+
+const render = createRender({ theme });
+
+const defaultProps = {};
+
+describe("/Page", () => {
```

Comment:
```suggestion
describe("<Page />", () => {
```
JSX syntax for components.